src/NumericalAlgorithms/LinearSolver/ConjugateGradient/ResidualMonitorActions.hpp

```diff
@@ -171,7 +173,9 @@ struct UpdateResidual {
     if (UNLIKELY(has_converged and
                  static_cast<int>(get<::Tags::Verbosity>(box)) >=
                      static_cast<int>(::Verbosity::Quiet))) {
-      Parallel::printf("%s", has_converged);
+      Parallel::printf("The linear solver has converged in %d iterations: %s",
```

Comment:
should this have format string `%llu` or similar for the size_t ?

Reply:
Changed to `%zu` now. I don't know exactly what the best-practice here is, so if you do please tell me.

Reply:
yes, `%zu` is what I use